src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1597,11 +1600,19 @@ def _ssl_context():
     return ssl.create_default_context()
 
 
-def _build_app_settings_output(app_settings, slot_cfg_names):
+def _build_app_settings_output(app_settings, slot_cfg_names, redact=False):
     slot_cfg_names = slot_cfg_names or []
     return [{'name': p,
              'value': app_settings[p],
-             'slotSetting': p in slot_cfg_names} for p in _mask_creds_related_appsettings(app_settings)]
+             'slotSetting': p in slot_cfg_names} for p in (_redact_appsettings(app_settings) if redact
+                                                           else _mask_creds_related_appsettings(app_settings))]
+
+
+def _redact_appsettings(settings):
+    logger.warning('App settings have been redacted. Use `az xxx config appsettings list` to view.')
```

Comment:
It is better to specify the specific command to users instead of using `xxx`.

Reply:
The method covers three types of apps—webapp, logicapp, and functionapp, so the placeholder is there to avoid specifying which one it is.

Reply:
Suggest specifying the clear commands, otherwise it may easily confuse customers
```suggestion
    logger.warning('App settings have been redacted. Use `az webapp/logicapp/functionapp config appsettings list` to view.')
```